libafl_qemu/src/modules/mod.rs

```diff
@@ -18,7 +18,7 @@ pub mod systemmode;
 pub use systemmode::*;
 
 pub mod edges;
-pub use edges::EdgeCoverageModule;
+pub use edges::*;
```

Comment:
Not a fan of blanked imports tbh, there's even a clippy lint for that (and it can autofix)

Reply:
ah cool didn't know about this, could be good to do it

Reply:
@domenukk should that be part of `scripts/autofix.sh`? Because when I ran that, it didn't change it.

Reply:
i'll solve this in another pr, we have other pub use of this kind